integration-tests/src/lib.rs

```diff
@@ -11,3 +11,79 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
```

Comment:
Please move these helpers into the `benchmarks` module (create a `benchmarks/mod.rs` and just define it here as `pub mod benchmarks`.

Because this is the root module for integration-tests and benchmarks together.